src/watch/fileWatchers.ts

```diff
@@ -49,14 +53,17 @@ export default class FileWatcher {
 	fileExists: boolean;
 	fsWatcher: FSWatcher | fs.FSWatcher;
 	tasks: Set<Task>;
+	transformDependencyTasks: Set<Task>;
 
 	constructor(id: string, chokidarOptions: WatchOptions, dispose: () => void) {
 		this.tasks = new Set();
+		this.transformDependencyTasks = new Set();
 
-		let data: string;
+		let mtime = -1;
```

Comment:
I know this corresponds to the key of the fs.statSync response but I think calling this `modificationTime` would make the code below a little more readable to people not that familiar with this function.